anaconda.spec.in

```diff
@@ -201,6 +201,9 @@ Requires: gdb
 # support for installation from image and live & live image installations
 Requires: rsync
 Recommends: zram-generator-defaults
+# Display stuff moved from lorax templates
+Requires: xhost
+Requires: xorg-x11-xauth
```

Comment:
I see it in the sshd_config.anaconda file in lorax, but nowhere else. https://github.com/weldr/lorax/blob/master/share/templates.d/99-generic/config_files/common/sshd_config.anaconda

Reply:
Thanks! Vlada, could you add there a comment that `xorg-x11-xauth` is a dependency of the `anaconda-sshd` service?

Reply:
Thanks! Done.

Reply:
@bcl Is there any reason for allowing X11Forwarding in sshd_config.anaconda?
This was needed in the past when it was possible to execute graphical installation via X11 forwarding on s390x, but this is not supported/possible since RHEL-7 any more.

Reply:
After a more careful look, I'm no longer sure about where `xauth` goes (`-img-` or `-env-`). Earlier, I got "it's used by liveinst setup script" but now I see it in `sshd_config.anaconda`, which is used by `runtime-postinstall.tmpl`, which is documented as altering the installer environment.

Reply:
@jstodola I doubt it's useful anymore, and looking at the Anaconda logs it appears this was first added in 2004 for s390. Things have changed a whole bunch since then ;)

Reply:
The `xauth` package is now removed here. I've updated the lorax template changes to do that too: https://github.com/weldr/lorax/pull/1125